src/System.Security.Cryptography.Algorithms/tests/AesTests.cs

```diff
@@ -0,0 +1,134 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Security.Cryptography.Algorithms.Tests
+{
+    public partial class AesTests
+    {
+        [Fact]
+        public static void AesDefaultCtor()
+        {
+            using (Aes aes = new AesMinimal())
+            {
+                Assert.Equal(256, aes.KeySize);
+                Assert.Equal(128, aes.BlockSize);
+                Assert.Equal(CipherMode.CBC, aes.Mode);
+                Assert.Equal(PaddingMode.PKCS7, aes.Padding);
+            }
+        }
+
+        private class AesMinimal : Aes
+        {
+            // If the constructor uses a virtual call to any of the property setters
+            // they will fail.
+            private readonly bool _ready;
+
+            public AesMinimal()
+            {
+                // Don't set this as a field initializer, otherwise it runs before the base ctor.
+                _ready = true;
```

Comment:
The real world example involves an AES type which is acting as a wrapper object, actually :smile:.

They create their worker object in the constructor (vs field initializer) and deferred the get/set to it.  Since the wrapped object hadn't been constructed yet the call to set_KeySize resulted in a nullref.

This test emulates that with a boolean instead of a nullref.